airbyte-config/config-models/src/main/java/io/airbyte/config/StreamResetRecord.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config;
+
+import java.util.UUID;
+import javax.annotation.Nullable;
+
+public record StreamResetRecord(UUID connectionId,
+                                String streamName,
+                                @Nullable String streamNamespace) {
```

Comment:
updated this - let me know if it looks right!

Reply:
I looks right. I don't have anything against using record but in the other parts of the codebase we are using lombok because it is more customizable. I wonder if it should be the same here:
```
@Data
public class StreamResetRecord() {
private final @NonNull UUID connectionId,
private final @NonNull String streamName,
private final String streamNamespace
}
```